lib/Wikia/src/Service/User/Preferences/PreferenceService.php

```diff
@@ -2,26 +2,243 @@
 
 namespace Wikia\Service\User\Preferences;
 
-use Wikia\Domain\User\GlobalPreference;
+use Doctrine\Common\Cache\CacheProvider;
+use Wikia\Domain\User\Preferences\LocalPreference;
+use Wikia\Domain\User\Preferences\UserPreferences;
+use Wikia\Logger\Loggable;
+use Wikia\Persistence\User\Preferences\PreferencePersistence;
+use Wikia\Util\WikiaProfiler;
 
-interface PreferenceService {
+class PreferenceService {
```

Comment:
Should this still satisfy an interface? It would make the consumers of this more testable.